composer.json

```diff
@@ -12,11 +12,17 @@
         }
     ],
     "require": {
-        "roundcube/plugin-installer": ">=0.1.6"
+        "php": "^5.4 || ^7.0 || ^8.0",
+        "roundcube/plugin-installer": "^0.3.2"
```

Comment:
Older versions of Roundcube might ship with older versions of the plugin installer (IIRC the "complete" dist includes it), so we might want to accept older versions too?

Reply:
Indeed: https://github.com/roundcube/roundcubemail/blob/1.3.0/composer.json-dist#L25

Reply:
Fixed with https://github.com/GermanCoding/Roundcube_TLS_Icon/pull/7/commits/a48684dec111f8317363d17f5bb785971bc0842f